.rubocop.yml

```diff
@@ -1158,4 +1158,4 @@ Style/WordArray:
   Enabled: false
 
 Style/FrozenStringLiteralComment:
-  Enabled: true
+  Enabled: false
```

Comment:
was to fix error in Codeclimate `Missing frozen string literal comment` @hallelujah should approve though